subsys/storage/flash_map/flash_map.c

```diff
@@ -187,15 +187,19 @@ int flash_area_get_sectors(int idx, uint32_t *cnt, struct flash_sector *ret)
 int flash_area_read(const struct flash_area *fa, off_t off, void *dst,
```

Comment:
Changes to flash area should not be part of this PR.

Reply:
done